samples/scripts/build-logs

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+if [ $# -eq 1 ]; then
+    BUILD_RUN_NAME=$1
+    FOLLOW=0
+elif [ $# -eq 2 ] && [ "$2" = "--follow" ]; then
+    BUILD_RUN_NAME=$1
+    FOLLOW=1
+else
+    echo "Usage: build-logs <BUILD_RUN_NAME> [--follow]"
+    exit 1
+fi
+
+BUILD_RUN_NAME=$1
+
+# Retrieve the buildrun
+
+BUILD_RUN=`kubectl get buildrun ${BUILD_RUN_NAME} -o json 2>/dev/null`
+
+if [ $? -ne 0 ]; then
+    echo "A build run with the name '${BUILD_RUN_NAME}' cannot be found."
+    exit 2
+fi
+
+echo "Build run: ${BUILD_RUN_NAME}"
+
+# Extract the task run name from the buildrun
+
+set -e
+
+TASK_RUN_NAME=`echo ${BUILD_RUN} | jq -r '.status.latestTaskRunRef'`
+
+echo "Task run: ${TASK_RUN_NAME}"
+
+# Find the pod, it has labels for the build run and task run name
+
+PODS=`kubectl get pods -l "buildrun.build.dev/name=${BUILD_RUN_NAME},tekton.dev/taskRun=${TASK_RUN_NAME}" -o json`
+PODS_LENGTH=`echo ${PODS} | jq ".items | length"`
+
+if [ ${PODS_LENGTH} -eq 0 ]; then
+    echo "No pod found. There is probably a problem in your build configuration."
+    exit 3
+elif [ ${PODS_LENGTH} -ne 1 ]; then
```

Comment:
should be `elif [ "${PODS_LENGTH}" -ne 1 ]; then`, see https://github.com/koalaman/shellcheck/wiki/SC2086

Reply:
Done.